inst/BS5/assets/pkgdown.scss

```diff
@@ -20,6 +20,17 @@
 
 /* navbar =================================================================== */
 
+$pkgdown-navbar-bg: null !default;
+
+// BS navbars appears to be designed with the idea that you have a coloured
+// navbar that looks the same in both light and dark mode. We prefer a mildly
+// coloured navbar that's just different enough from the body to stand out.
+.navbar {
+  background: RGBA(var(--bs-body-color-rgb), 0.1);
+  background: color-mix(in oklab, color-mix(in oklab, var(--bs-body-bg) 95%, var(--bs-body-color)) 98%, var(--bs-primary));
+  background: $pkgdown-navbar-bg;
+}
+
```

Comment:
Also the `.nav-link` background styles in the lines just below this could use the transparency or `color-mix()` strategy as well, something like this would work:


```css
.nav-item.active .nav-link       { background: RGBA(var(--bs-body-color-rgb), 0.1); }
.nav-item        .nav-link:hover { background: RGBA(var(--bs-body-color-rgb), 0.25); }
```

Reply:
Ah good idea, I should use the same technique for the navbar and the TOC.

But I think defining the css variables here is a bit more trouble than it's worth. I think we can take a pass at that again in a couple of years when we can just use `color-mix()` unconditionally.